src/GraphQL.MicrosoftDI/ScopedFieldBuilderExtensions.cs

```diff
@@ -9,18 +9,37 @@ namespace GraphQL.MicrosoftDI;
 /// </summary>
 public static class ScopedFieldBuilderExtensions
 {
+    /// <summary>
+    /// Applies scoped middleware to the field. A dependency injection scope is created for the duration of the field resolver's execution
+    /// and the scoped service provider is passed within <see cref="IResolveFieldContext.RequestServices"/>.
+    /// </summary>
+    [AllowedOn<IObjectGraphType>]
+    internal static FieldBuilder<TSourceType, TReturnType> Scoped<TSourceType, TReturnType>(this FieldBuilder<TSourceType, TReturnType> builder)
```

Comment:
Currently internal.  Could be public, but we already have ResolveScoped plus the Resolve() builder methods such as `Resolve().WithScope().WithService<T>` etc